var/spack/repos/builtin/packages/umpire/package.py

```diff
@@ -458,37 +456,56 @@ def initconfig_package_entries(self):
     def cmake_args(self):
         return []
 
-    def test(self):
+    def run_umpire(self, exe, expected):
```

Comment:
This runs a different test each time so a name along the lines `run_test` is more suitable.

Reply:
🤦 Apologies.  I did not phrase that correctly.  I meant `check_test`...